packages/web/pages/justread/index.tsx

```diff
@@ -281,6 +290,37 @@ const SubscriptionSourceHoverContent = (
     return undefined
   }, [subscriptions])
 
+  const sendHomeFeedback = useCallback(
+    (feedbackType: SendHomeFeedbackType) => {
+      ;(async () => {
```

Comment:
Why do we need IIFE here?

Reply:
Ah i guess this used to be needed, not so much now, you can make useCallback async